### PR TITLE
Adding Stop Button to the AudioPlayer

### PR DESCRIPTION
**Pull request by bahaa96**
### Title of change
Adding Stop Button in Player. #673 

### Checklist

- [ ] Unit tests written
- [ ] Manually tested
- [ ] Prettier & ESLint were run
- [ ] New dependencies are included in `package-lock.json`

### Screenshot

Something Else: 
After Submitting this issue #991 I've tried to fix this #525 after mocking your db but I failed to get around the audio player in a day or specifically When I click "Next Surah" The Media Player Starts from The verse "2:2" not "2:1".

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32823

**Comment by bahaa96** [comment]
Tha .mp3 files URL is wrong at the staging API

**Comment by sabeurthabti** [comment]
@bahaa96 thanks for taking time to contribute. 

I am not sure an extra button is somethign that needed - from my understanding the play button becomes a stop button.  

cc @mmahalwy @naveed-ahmad

File: .env [hunk 1]
```diff
@@ -8,3 +9,5 @@ SENTRY_KEY_SERVER=
 # Quran.com - development app, no need to worry!
 FACEBOOK_APP_ID=1599019233731707
 FONTS_URL=//quran-1f14.kxcdn.com
+PWD=D:\Projects\quran.com-frontend
+pm_cwd=D:\Projects\quran.com-frontend
```
[the remark this answers is not in the document]
Comment (bahaa96) [reply .env]: Yes You can remove them.

**Comment by bahaa96** [comment]
OK but how tha user would get back to tha first ayah If he didnt finish tha Surah yet

**Comment by bahaa96** [comment]
tha first solution I was about to make tha the forward button would navigate to the next Surah, like any music app. But you're using the forward and backward buttons to navigate between Ayahs Not Surahs.
Tell me If you get better solution. <3

File: .env (hunk 1 above)
[the remark this answers is not in the document]
Comment (bahaa96) [reply .env]: I'll submit a PR to to add Instructions to add these two vars into .env file cause they're important and being used at server.js. But feel free as they weren't at the repo when I first cloned it.

**Comment by bahaa96** [comment]
Tell me tha last decision and I'll change the button immediately. It's hard to compare your audio player with Spotify's cause Spotify Plays a whole song not verses, so they use backward button to get back to tha first of tha song but in our case I cant get an idea to make link Spotify, either letting the user to change the Ayah number from the media player or to make the "Back to Beginning" button float.

**Comment by naveed-ahmad** [comment]
Stop button make sense, clicking on it should stop the player and reset the time to start of currently playing ayah( like standard players). No need to go back to first ayah of surah, that'll be very confusing.

File: .env [hunk 2]
```diff
@@ -8,3 +8,4 @@ SENTRY_KEY_SERVER=
 # Quran.com - development app, no need to worry!
 FACEBOOK_APP_ID=1599019233731707
 FONTS_URL=//quran-1f14.kxcdn.com
+
```
Comment (naveed-ahmad) [inline .env]: please remove empty line as well

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32828

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32830

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32831

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32832

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32833

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32834

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32835

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32836

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32837

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32838

**Pull request closed without merging**